### PR TITLE
[#324] Add details on how to run the build with Podman

### PR DESCRIPTION
**Pull request by DavideD**

### CODE REVIEW

**Comment by blafond** [comment]
So running through the set-up and test with new instructions

1) Wouldn't it be helpful to add a note on what to expect when a user runs "./gradle test" ?

I'm getting:

Configure project :
Parsing ORM version
Hibernate ORM Version: 5.4.22.Final
Vert.x Version: 3.9.3

BUILD SUCCESSFUL in 1s
11 actionable tasks: 11 up-to-date

2) noticed you changed the command to **./gradlew build** rather than **./gradlew test** ... guessing they are the same in this case?

**Comment by DavideD** [comment]
> Wouldn't it be helpful to add a note on what to expect when a user runs "./gradle test" ?

I don't know. Apart for the versions of the libraries, everything else is the standard  Gradle output.
Once you've reached the `BUILD SUCCESSFUL` message. was there something not clear about them?
They might also change in the future and I would prefer not having to update the file each time.
These files are already quite big and they will become bigger with the addition of new datastores.

> noticed you changed the command to **./gradlew build** rather than **./gradlew test** ... guessing they are the same in this case?

They are basically the same but you are right. We used test everywhere else and I prefer to be consistent.
I will change it.

File: podman.md [hunk 1]
```diff
@@ -1,61 +1,110 @@
-# Running Hibernate Reactive tests on Podman
+# How to start the test databases using Podman
 
-As an alternative to Docker, you can use [podman] to start the databases used 
-by the test suite for Hibernate Reactive.
+The following sections will use [Podman][podman] to start the database of your choice, create the schema
+and create the credentials required to run the tests during the build.
+
+Podman is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System.
+Containers can either be run as root or in rootless mode.
```
Comment (gavinking) [inline podman.md:7]: A minor comment: `.md` files are supposed to be readable as text, so it's important to maintain a consistent line-length.

**Review by gavinking — CHANGES_REQUESTED** [review]

File: podman.md (hunk 1 above)
Comment (DavideD) [reply podman.md:7]: ok

**Comment by DavideD** [comment]
I've applied the remarks. I'm going to merge it if there isn't anything else.

File: README.md [hunk 2]
```diff
@@ -160,13 +160,10 @@ directory.
 
 #### If you have Podman
 
-If you use [Podman][], you can start the test database by following 
+If you have [Podman][podman] installed, you can start the test database by following
```
Comment (gavinking) [inline README.md:163]: @DavideD I know it's picky, but this line is also really long compared to the rest of the README

**Review by gavinking — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (DavideD) [reply README.md]: Should be fixed now

**Review by gavinking — APPROVED** [review]

**Comment by DavideD** [comment]
Merged https://github.com/hibernate/hibernate-reactive/pull/404

Thanks

**Pull request merged**